vgsshow/src/main/java/com/verygoodsecurity/vgsshow/core/network/headers/StaticHeadersStore.kt

```diff
@@ -23,14 +23,14 @@ internal interface StaticHeadersStore {
     fun remove(key: String)
 
     /**
-     * @return all custom headers.
+     * @return all headers.
      */
     fun getAll(): Map<String, String>
 
     /**
-     * @return true if user add any header
+     * @return all headers what user set.
      */
-    fun containsUserHeaders(): Boolean
```

Comment:
Restore please `containsUserHeaders`. If someone already use it, then it will break the existing solution of the client.

Reply:
 @Endorf Thank you for your attention, but this function exists  inside the internal class so It can't be used outside the library